scripts/README.md

```diff
@@ -121,32 +110,8 @@ paths:
       rule: ""
       channel-list: []
 ```
-* Add a section to hermes
-```
-[[chains]]
-id = '{CHAIN_ID}'
-rpc_addr = 'http://{NODE_PREFIX}1:26657'
-grpc_addr = 'http://{NODE_PREFIX}1:9090'
-websocket_addr = 'ws://{NODE_PREFIX}:26657/websocket'
-rpc_timeout = '10s'
-account_prefix = '{ADDRESS_PREFIX}'
-key_name = 'hrly{next relayer ID}'
-store_prefix = 'ibc'
-default_gas = 100000
-max_gas = 3000000
-gas_price = { price = 0.000, denom = '{DENOM}' }
-gas_multiplier = 1.1
-max_msg_num = 30
-max_tx_size = 2097152
-clock_drift = '5s'
-max_block_time = '10s'
-trusting_period = '119s'
-trust_threshold = { numerator = '1', denominator = '3' }
-address_type = { derivation = 'cosmos' }
-```
-* Finally add the execution of the `init_chain` script for this host zone in `scripts/start_network.sh`, and add it to the array of `HOST_CHAINS`
-```
-sh ${SCRIPT_DIR}/init_chain.sh {NEW-HOST-ZONE}
+* Finally add the host zone to the `HOST_CHAINS` array in `scripts/start_network.sh`
+```
 HOST_CHAINS=(GAIA JUNO OSMO ... {NEW-HOST-ZONE})
```

Comment:
Based on our testing setup and needing the different ibc denoms, I think we're limited to running 4 zones (IIRC)? If that's the case, suggest adding a comment

Reply:
Ah yes, good call! Will add